recipes-bsp/partition/qcom-partition-confs_1.0.bb

```diff
@@ -0,0 +1,45 @@
+SUMMARY = "Qualcomm machine specific partition configuration"
+DESCRIPTION = "Machine partition configuration used by qcom-gen-partitions-tool \
+for generating the machine specific partition.xml file."
+LICENSE = "BSD-3-Clause-Clear"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause-Clear;md5=7a434440b651f4a472ca93716d01033a"
+
+DEPENDS += "qcom-gen-partitions-tool-native"
+
+SRC_URI = " \
+    file://qcm6490-partitions.conf \
+    file://qcs9100-partitions.conf \
+"
+
+PROVIDES += "virtual/partconf"
+
+S = "${WORKDIR}/sources"
+UNPACKDIR = "${S}"
+B = "${WORKDIR}/build"
+
+inherit python3native deploy
+
+do_configure[noexec] = "1"
+
+PARTCONF ?= ""
+PARTCONF:qcm6490 ?= "qcm6490-partitions.conf"
+PARTCONF:qcs9100 ?= "qcs9100-partitions.conf"
+
+do_compile() {
+    # Generate partition.xml using gen_partition utility
+    ${PYTHON} ${STAGING_BINDIR_NATIVE}/gen_partition.py \
```

Comment:
with the `python3native` we can call the tools without the full path, so this can be just gen_partition.py

Reply:
anyway I will check locally to confirm

Reply:
confirmed, call just the tool works
```patch
-    ${PYTHON} ${STAGING_BINDIR_NATIVE}/gen_partition.py \
+    gen_partition.py \
```

Reply:
Updated.